### DISCUSSION TITLE
Can I integrate rocket chat inside an existing android app built in native code ?

### DISCUSSION THREAD

**Discussion by bisverse** [RocketChat/Rocket.Chat.ReactNative#5164] [General]
I want integration into my existing app. Is it possible? How difficult will it be?

**Comment by dnlsilva** [comment]
If you only want the chat part use webview which works.
https://developer.rocket.chat/customize-and-embed/embedded-layout

Natively there is no support and the app made in react native does not have reusable components to be used in other apps.
